launchdarkly/client.go

```diff
@@ -28,32 +29,32 @@ func (c *Client) Get(url string, expectedStatus []int) (interface{}, error) {
 }
 
 func (c *Client) GetInto(url string, expectedStatus []int, target interface{}) error {
-	_, response, err := c.execute("GET", url, nil, expectedStatus)
+	_, response, err := c.execute("GET", url, nil, expectedStatus, nil, 0)
 
 	json.Unmarshal(response, target)
 
 	return err
 }
 
 func (c *Client) Post(url string, body interface{}, expectedStatus []int, target interface{}) error {
-	_, response, err := c.execute("POST", url, body, expectedStatus)
+	_, response, err := c.execute("POST", url, body, expectedStatus, nil, 0)
 
 	json.Unmarshal(response, target)
 
 	return err
 }
 
-func (c *Client) Patch(url string, body interface{}, expectedStatus []int) ([]byte, error) {
-	_, response, err := c.execute("PATCH", url, body, expectedStatus)
+func (c *Client) Patch(url string, body interface{}, expectedStatus []int, statusToRetry []int, numberOfRetry int) ([]byte, error) {
+	_, response, err := c.execute("PATCH", url, body, expectedStatus, statusToRetry, numberOfRetry)
 	return response, err
 }
 
 func (c *Client) Delete(url string, expectedStatus []int) error {
-	_, _, err := c.execute("DELETE", url, nil, expectedStatus)
+	_, _, err := c.execute("DELETE", url, nil, expectedStatus, nil, 0)
 	return err
 }
 
-func (c *Client) execute(method string, url string, body interface{}, expectedStatus []int) (int, []byte, error) {
+func (c *Client) execute(method string, url string, body interface{}, expectedStatus []int, statusToRetry []int, numberOfRetry int) (int, []byte, error) {
```

Comment:
Tu semble passer toujours les memes parametre, il y a une raison pourquoi ont voudrais _pas_ retry un certain call sur un 429? Si oui, pourquoi pas un boolean pour enabler le retry?

Reply:
Will used the parameter numberOfRetry as said boolean and remove statusToRetry